### ISSUE TITLE
ShareSheet do not open on iPad when openWithDefault: false

### ISSUE THREAD

**Issue by hydrococcous** [capacitor-community/file-opener#13]
**The bug**
If I set the openWithDefault option to false to open a file with an application program (e.g. *.docx with Word), the ShareSheet does not open or is not visible on the iPad (tested on iOS 15.5). But it works on the iPhone.

**System):**
 - iPadOS 15.5
 - ionic App
 - Capcitor 3

**Smartphone (please complete the following information):**
 - iPad 9th Generation

**Solution**
I changed the following line (74) in the FileOpenerPlugin.swift file
```
let rect = CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height);
```
to
```
let rect = CGRect(x: 0, y: 0, width: view.frame.width, height: 0);
```
I hope it helps if someone encounters this problem

**Comment by ryaa** [comment]
@hydrococcous thank you for the contribution.

I confirm the above problem and provided the required fix (see https://github.com/capacitor-community/file-opener/pull/15).
Can you please try to install the build with the fix and see if this works for you. To install please install the plugin by issuing the below command:
`npm install capacitor-community/file-opener#bugfix/13-sharesheet-do-not-open-on-ipad-when-openWithDefault-false-with-dist`

Now, on iPad, with **openWithDefault** set to false, there should be the ShareSheet showing in the center of the screen - see below
![Simulator Screen Shot - iPad (9th generation) - 2022-10-28 at 08 44 01](https://user-images.githubusercontent.com/3608222/198676610-e3311f48-95d7-4be2-a321-924553d2c9a0.png)

however, there is a new option parameter added **chooserPosition** and when this is provided the ShareSheet will be anchored to the button/point where the user tapped
![Simulator Screen Shot - iPad (9th generation) - 2022-10-27 at 17 43 13](https://user-images.githubusercontent.com/3608222/198677087-dd97e57e-06f0-44de-b0ec-5db586b97ed7.png)

Please see the updated README with the details [here](https://github.com/capacitor-community/file-opener/tree/bugfix/13-sharesheet-do-not-open-on-ipad-when-openWithDefault-false-with-dist)

Please note that I tested the fix on an iPad simulator only (since I don't have an iPad 9th generation) so I would really appreciate if you could test and let me know the results.

**Comment by ryaa** [comment]
The fix will be included into v1.0.3
